pkg/pipeline/output/base.go

```diff
@@ -1,22 +1,25 @@
 package output
 
 import (
+	"fmt"
+
 	"github.com/tinyzimmer/go-gst/gst"
 
 	"github.com/livekit/egress/pkg/config"
 	"github.com/livekit/egress/pkg/pipeline/builder"
+	"github.com/livekit/egress/pkg/types"
 )
 
 type outputBase struct {
 	audioQueue *gst.Element
 	videoQueue *gst.Element
 }
 
-func (b *Bin) buildOutputBase(p *config.PipelineConfig) (*outputBase, error) {
+func (b *Bin) buildOutputBase(p *config.PipelineConfig, egressType types.EgressType) (*outputBase, error) {
 	base := &outputBase{}
 
 	if p.AudioEnabled {
-		audioQueue, err := builder.BuildQueueWithLatency("audio_out_queue", p.Latency, true)
+		audioQueue, err := builder.BuildQueueWithLatency(fmt.Sprintf("audio_%s_queue", egressType), p.Latency, true)
```

Comment:
is this for debugging ? or that they have to be unique?

Reply:
just for debugging, gstreamer will just do queue0, queue1, etc